packages/atomic/src/pages/examples/folding.html

```diff
@@ -76,6 +86,365 @@
           <atomic-layout-section section="results">
             <atomic-smart-snippet></atomic-smart-snippet>
             <atomic-folded-result-list image-size="small">
+              <atomic-result-template must-match-sourcetype="Slack">
+                <template>
+                  <style>
+                    div.result-root.with-sections.display-list.density-normal atomic-result-section-visual {
+                      margin-right: 16px;
+                    }
+
+                    .slack-section-badge {
+                      align-items: center;
+                      display: flex;
+                      width: 100%;
+                    }
+
+                    .slack-title-icon {
+                      display: flex;
+                      width: 100%;
+                      margin-top: 8px;
+                    }
+
+                    .slack-badge::part(result-badge-element) {
+                      background-color: #86d8ff;
+                      color: white;
+                      border-radius: 32px;
+                      padding-top: 2px;
+                      padding-bottom: 2px;
+                    }
+
+                    ::part(children-root) {
+                      margin-left: 70px;
+                      border-left: thin solid #bcc3ca;
+                    }
+
+                    div.result-root atomic-result-image {
+                      height: 30px;
+                      width: 30px;
+                      margin-right: 16px;
+                      border-radius: 4px;
+                    }
+
+                    .slack-atomic-result-icon {
+                      flex-shrink: 0;
+                      height: 32px;
+                      width: 32px;
+                      background-size: cover;
+                      margin-right: 16px;
+                    }
+
+                    .slack-msg-result-section-title {
+                      display: flex;
+                      background-color: #f6f7f9;
```

Comment:
Instead of `#f6f7f9`, I would recommend using the variable with the same value `var(--atomic-neutral-light)`